internal/kafka/internal/handlers/admin_kafka.go

```diff
@@ -169,13 +171,33 @@ func (h *adminKafkaHandler) Update(w http.ResponseWriter, r *http.Request) {
 				return false
 			}
 
+			getStatusBasedOnSuspendedParam := func(susp *bool, kafka *dbapi.KafkaRequest) string {
+				if shared.IsNilPredicate(susp) {
+					return kafka.Status
+				} else {
+					if *susp {
+						if kafka.Status != constants.KafkaRequestStatusSuspended.String() {
+							return constants.KafkaRequestStatusSuspending.String()
+						}
```

Comment:
Running this with my kas-fleetshard changes, the status remains as `suspending`. I think this might be missing a transition to `suspended` somewhere.

Reply:
This is taken care in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1346